corehq/apps/app_manager/views/releases.py

```diff
@@ -340,21 +340,16 @@ class AppDiffView(LoginAndDomainMixin, BasePageView, DomainViewMixin):
     template_name = 'app_manager/app_diff.html'
 
     @use_angular_js
-    @login_and_domain_required
     def dispatch(self, request, *args, **kwargs):
-        try:
-            self.first_app_id = self.kwargs["first_app_id"]
-            self.second_app_id = self.kwargs["second_app_id"]
-            self.first_app = Application.get(self.first_app_id)
-            self.second_app = Application.get(self.second_app_id)
-            if not (request.couch_user.is_member_of(self.first_app.domain)
-                    and request.couch_user.is_member_of(self.second_app.domain)):
-                raise PermissionDenied()
```

Comment:
@emord I think this check got lost? That means you can diff with an app in any domain without being a superuser or member of the other domain. I'm going to add it back.